### PR TITLE
Fix of undifined offset

### PR DESCRIPTION
**Pull request by Vladnev**
Some servers return not valid first header.
Fix according to issue #1909

### CODE REVIEW

File: src/Handler/EasyHandle.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function createResponse()
 
         // Attach a response to the easy handle with the parsed headers.
         $this->response = new Response(
-            $startLine[1],
+            isset($startLine[1]) ? (integer) $startLine[1] : 200,
```
Comment (GrahamCampbell) [inline src/Handler/EasyHandle.php]: Should be `(int)` rather than `(integer)`.

Comment (GrahamCampbell) [reply src/Handler/EasyHandle.php]: Also, rather than 200, doesn't Guzzle have an http ok constant that could be used?

Comment (Vladnev) [reply src/Handler/EasyHandle.php]: Fixed to (int) - about const - I did not found any statuses constants.
But I don't sure is better to stay 200 status or maybe we can have some better variants

Comment (sagikazarmark) [reply src/Handler/EasyHandle.php]: No constants AFAIK, just this list: https://github.com/guzzle/psr7/blob/master/src/Response.php#L15

**Comment by sagikazarmark** [comment]
Sorry for the huge delay.

I don't think it's a good idea to return 200 automatically. That's something one simply cannot assume.

The starting line is a key part of the HTTP message, if it's not valid then I believe we cannot safely return a response. That would produce unreliable behavior, good luck debugging that.

**Comment by FrontEndCoffee** [comment]
I think this should be closed, or a Parsing Exception should be thrown in the case of an invalid first header.

This would be more clear to developers IMHO.

**Comment by sagikazarmark** [comment]
I would be happy to accept a PR implementing the suggested solution. Until then, closing this.

**Pull request closed without merging**